pom.xml

```diff
@@ -103,9 +104,19 @@
     <repositories>
         <repository>
             <id>snapshot-repository</id>
-            <name>Maven2 Snapshot Repository</name>
+            <name>Hazelcast Snapshot Repository</name>
```

Comment:
This is not Hazelcast repository, it's maven central repo for snapshot

Reply:
This came from https://github.com/hazelcast/hazelcast-wm/commit/9fb91a4ae7a939c4db49d2d87a041dd71c33329f. So @Fly-Style, can you weigh in?